discord/guild_const.go

```diff
@@ -49,38 +53,85 @@ const (
 	Banner GuildFeature = "BANNER"
 )
 
-type ExplicitFilter uint8
+// ExplicitFilter is the explicit content filter level of a guild.
+type ExplicitFilter nullable.Enum
 
-const (
-	NoContentFilter ExplicitFilter = iota
-	MembersWithoutRoles
-	AllMembers
+var (
```

Comment:
Above structs already use `*ExplicitFilter`, do we need a `nullable.Enum` here? Also why var?

Reply:
[Docs](https://discord.com/developers/docs/resources/guild#modify-guild) say the field is nullable, which I assume triggers a reset to default.

Reply:
For var see above comment